mobile/translations/ressources/en.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  homePage: {
+    title: "Home page"
+  },
+  error: {
+    message: "An error occured: "
```

Comment:
How is this supposed to be used, if you want to use this correctly I think it would be better to be able to get params inside where you'll be able to get directly the error message. 

```javascript
message: (errorMessage) => `An error occurred: ${errorMessage}`,
```

Reply:
Note: Didn't end up doing this because if not we would have to call a function inside a string